content/modules/upgrading-rs.md

```diff
@@ -70,27 +70,31 @@ When you upgrade the module for a database, you can either:
 1. To upgrade the module for the database, run:
 
     ```src
-    rladmin upgrade module db_name <database_name> module_name <module_name> version <new_module_version_number> module_args <module arguments>
+    rladmin upgrade db <database_name> and module module_name <module_name> version <new_module_version_number> module_args "<module arguments>"
     ```
 
+    - If you want to remove the existing module arguments, enter `""` without arguments.
+    - If you want to use the exisintg module arguments, enter `"keep_args"`
+    - If you want to update multiple modules, use the `and module` parameter for each module.
```

Comment:
its not only for multiple modules but also for one module you will need "and module" (-i.e. in addition to the db). My comment does necessarily requires changes